R/utils_memoised.R

```diff
@@ -69,8 +69,6 @@ memoise_bioc_mirrors <- memoise::memoise({
 
 #' @importFrom memoise memoise
 memoise_available_packages <- memoise::memoise({
-  Sys.sleep(20)
-
```

Comment:
This one's embarrassing. `git blame` reveals that this is residual from noneother than myself. I don't know exactly why I added it, but I assume it was to test the memoisation. No better time than the present to fix it.